CONTRIBUTION.rst

```diff
@@ -0,0 +1,532 @@
+
+    # Pour travailler, nous aurons besoin d'une branche, basée sur une version à jour
+    # (fraîchement récupérée) de la branche upstream/3.8. On met donc à jour notre
+    # version locale.
+    git fetch upstream
+
+    # On créé ensuite une branche. Il est pratique de nommer la branche en fonction du
+    # fichier sur lequel on travaille. Par exemple, si vous travaillez sur
+    # « library/venv.po », vous pouvez nommer votre branche « library-venv ».
+    # Cette nouvelle branche nommée « library-sys » est basée sur « upstream/3.8 ».
+    git checkout -b library-sys upstream/3.8
+
+    # Vous pouvez maintenant travailler sur le fichier (typiquement, en utilisant poedit).
+    # N'oubliez pas de configurer poedit pour passer à la ligne à 79 caractères.
+    # (Édition -> Préférences -> Avancé -> Passer à la ligne à : 79)
+    # Ici, remplacez « library/sys.po » par le fichier que vous avez choisi précédemment.
+    poedit library/sys.po
+
+    # Si vous n'utilisez pas poedit, vous pouvez utiliser `powrap` (voir la section *outils*)
+    # qui va reformater correctement le fichier que avez vous avez modifié.
+    # Exécutez `powrap -m` (reformater tous les fichiers modifiés)
+    # ou `powrap library/sys.po` (un fichier en particulier) :
+    powrap -m
+
+    # Pour l'orthographe, une liste blanche de certains termes techniques ou
+    # de noms propres, comme « Guido », « C99 » ou « sérialisable », est
+    # stockée dans le fichier « dict » à la racine du projet.  Vous pouvez
+    # bien sûr y ajouter une entrée si nécessaire.
+    make verifs
+
+    # C'est le moment de git add et git commit
+    # C'est l'occasion de se relire avec l'option -p ou -u.
+    git add library/sys.po
+
+    git commit -m "Traduction de library/sys.po"  # Ou un autre message plus inspiré :)
+
+    # Poussez ensuite vos modifications sur votre fork Github.
+    # Le -u n'est utile qu'une fois pour que votre client git se souvienne que cette
+    # branche est liée à votre fork Github (et donc qu'un futur `git pull` sache quoi
+    # tirer)
+    git push -u origin
+
+    # La commande précédente vous affichera un lien pour ouvrir une pull request sur
+    # Github. Si vous l'avez manqué, allez simplement sur https://github.com/python/python-docs-fr/pulls
+    # et un joli bouton « Compare & pull request » devrait apparaître au bout de
+    # quelques secondes vous indiquant que vous pouvez demander une pull request.
+
+    # À partir de là, quelqu'un passera en revue vos modifications, et fera desvous voudrez
+    # suggestions/corrections. Pour les prendre en compte, retournez sur votre branche
+    # contenant du fichier concerné (au cas où vous auriez commencé quelque chose d'autre
+    # sur une autre branche) :
+    git checkout library/sys
+    git pull  # pour rapatrier les modifications que vous auriez accepté
+              # sur l'interface web.
+
+    # Réglez les problèmes, puis commitez à nouveau :
+    git commit -a -m "sys : corrections".
```

Comment:
```suggestion
    git commit -a -m "prise en compte des remarques"
```

À voir pour le message. Par contre il y a un point en trop à la fin de la phrase.

Reply:
Après de toute façons on `squash and merge` les PRs, donc ce message va disparaître.

Reply:
@JulienPalard Je maintiens la correction de @vpoulailleau en tant que première version.